packages/thirdweb/src/exports/thirdweb.ts

```diff
@@ -188,6 +188,50 @@ export type {
   PayTokenInfo,
 } from "../pay/utils/commonTypes.js";
 
+/**
+ * CHAINSAW
+ */
+export {
+  getTransactions,
+  type GetTransactionsResult,
+  type GetTransactionsParams,
+} from "../chainsaw/endpoints/getTransactions.js";
+
+export {
+  getNFTsByOwner,
+  type GetNFTsByOwnerParams,
+  type GetNFTsByOwnerResult,
+} from "../chainsaw/endpoints/getNFTsByOwner.js";
+
+export {
+  getNFTsByCollection,
+  type GetNFTsByCollectionParams,
+  type GetNFTsByCollectionResult,
+} from "../chainsaw/endpoints/getNFTsByCollection.js";
+
+export {
+  getBlock,
+  type GetBlockParams,
+} from "../chainsaw/endpoints/getBlock.js";
+
+export {
+  getEvents,
+  type GetEventsParams,
+  type GetEventsGroupBy,
+  type GetEventsInterval,
+  type GetEventsResult,
+} from "../chainsaw/endpoints/getEvents.js";
+
+export {
+  getLatestBlockNumber,
+  type GetLatestBlockNumberParams,
+} from "../chainsaw/endpoints/getLatestBlockNumber.js";
+
+export type {
```

Comment:
Worried this will get confusing with our existing transaction types, not sure what to do yet

Reply:
I prefixed all the types that Chainsaw functions return with `Chainsaw`. 
So we have `ChainsawBlock`, `ChainsawEvents`, `ChainsawTransactions` and `ChainsawNFTs`
Not the most elegant solution, but it's more clear

Reply:
can we unify the return types? is there a reason that return types (not input params) have to be different than existing `block` `transaction` etc types?